capstone-app/src/views/ProfilePage/Sections/Journal.js

```diff
@@ -0,0 +1,90 @@
+
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField } from "@material-ui/core";
+
+import Button from "components/CustomButtons/Button.js";
+import React from 'react';
+
+import styles from "assets/jss/material-kit-react/javascriptStyles.js";
+
+/** Survey component to collect PANAS survey data. */
```

Comment:
You should write your own documentation (throughout everything in this PR) - this is from my code.